umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/monitoring.md

```diff
@@ -1,8 +1,76 @@
 ---
 title: Monitoring & Management
 menuTitle: Monitoring & Management
-description:
+description: Monitor and manage both the Data and the Device & Container
+  Infrastructures using the Management Console.
 weight: 2000
 edition: community
-draft: true
+draft: false
 ---
+
+The Management Console supports you to monitor and manage the Data Infrastructure 
+and the Device & Container Infrastructure.
+
+## When should I use it?
+
+Once initial deployment of the United Manufacturing Hub is completed, you can 
+monitor and manage it using the Management Console. If you have not deployed yet, 
+navigate to the [Get Started!](/docs/getstarted/installation/) guide.
+
+
+## What can I do with it?
+
+You can monitor the statuses of the following items using the Management Console:
+- **Modules**: A Module refer to a grouped set of related Kubernetes components 
+like pods, statefulsets, and services. It provides a way to monitor and manage 
+these components as a single unit.
+- **System**:
+    - Resource Utilization: CPU, RAM, and DISK usages.
+    - OS information: the used operating system, kernel version, and instruction 
+    set architecture.
+- **Data source**: the rate of Kafka/TimescaleDB messages per second, the health 
+of data sources.
+- **Data connection**: the health of connections. It checks the latency between 
+the UMH instance and the corresponding data source, for example, an OPC-UA server.
+- **Kubernetes**: the number of error events and the deployed management 
+companion's and UMH's versions.
+
+In addtion, you can check the topic structure used by data sources and the 
+corresponding payloads.
+
+Moreover, you can create a new connection and initilize the created connection to 
+deploy a data source.
+<!--Add delete and edit if these functionalities are available-->
+
+## How can I use it?
+
+After logging in, the **INSTANCE DASHBOARD** page shows the **Overview** tab. 
```

Comment:
Don't capitalize and don't use bold for Instance Dashboard, Only capitalize the first letter. This is because in the sentence you don't refer to the button but to the entity itself. Same thing would technically be for "Overview", but leave it bold as it shows that has more relevance in the context